plugins/modules/dcnm_network.py

```diff
@@ -1439,6 +1445,7 @@ def validate_input(self):
             deploy=dict(type='bool'),
             gw_ip_subnet=dict(type='ipv4_subnet'),
             vlan_id=dict(type='int', range_max=4094),
+            routing_tag=dict(type='int', default = 12345, range_max=4294967295),
```

Comment:
Please make the default consistent without spaces between the assignment `=`

`default=12345`

Reply:
done.